src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/CachePools/redis_config.yml

```diff
@@ -0,0 +1,22 @@
+imports:
+    - { resource: ../config/default.yml }
+
+services:
+    cache.adapter.redis.connection:
+        public: false
+        class: Redis
+        calls:
+            - [connect, [127.0.0.1]]
+
+    cache.adapter.default:
+        abstract: true
+        parent: cache.adapter.redis
+        tags:
+            - name: cache.pool
+              provider: cache.adapter.redis.connection
```

Comment:
please make it namespaced, to avoid clearing the whole local redis when running tests